GNUmakefile

```diff
@@ -393,6 +395,7 @@ CPPCHECK_ARGS ?=
 cppcheck:
 	cppcheck $(CPPCHECK_ARGS) -I include --max-configs=30	\
 		--library=posix --library=gnu --library=gtk	\
+		$(GLIB_CFLAGS) -D__GNUC__			\
 		--inline-suppr -q replace lib daemons tools
 
```

Comment:
Oh cool, I'd been meaning to ask you what your cppcheck command line looks like but I guess here it is.